res/values/strings.xml

```diff
@@ -237,6 +237,9 @@
     <string name="confirmation_remove_file_alert">"Do you really want to remove %1$s?"</string>
     <string name="confirmation_remove_folder_alert">"Do you really want to remove %1$s and its contents?"</string>
     <string name="confirmation_remove_local">Local only</string>
+    <string name="confirmation_remove_folder_local">Local only</string>
```

Comment:
These three lines should go away, they are obsolete; probably recovered while fixing a conflict in a rebase, right?